specification/components/schemas/HumanName.yaml

```diff
@@ -1,53 +1,15 @@
 type: object
-required:
-  - id
-  - use
-  - family
 properties:
-  id:
-    type: string
-    description: Unique object identifier for this name.
-    readOnly: true
-    example: '123'
-  use:
```

Comment:
We don't support 'use' for names although we do for addresses. Previous name is currently just the second occurrence of name.